galsim/utilities.py

```diff
@@ -418,6 +418,103 @@ def _gammafn(x):
                0.00000000000000000141, -0.00000000000000000023, 0.00000000000000000002
              )
 
+def deInterleaveImage(image, N, conserve_flux=False,suppress_warnings=False):
+    """
+    The routine to do the opposite of what 'interleaveImages' routine does. It generates a
+    (uniform) dither sequence of low resolution images from a high resolution image.
+
+    Many pixel level detector effects, such as interpixel capacitance, persistence, charge
+    diffusion etc. can be included only on images drawn at the native pixel scale, which happen to
+    be undersampled in most cases. Nyquist-sampled images that also include the effects of detector
+    non-idealities can be obtained by drawing multiple undersampled images (with the detector
+    effects included) that are offset from each other by a fraction of a pixel. If the offsets are
+    uniformly spaced, then images can be combined using 'interleaveImages' into a Nyquist-sampled
+    image.
+
+    Drawing multiple low resolution images of a light profile can be a lot slower than drawing a
+    high resolution image of the same profile, even if the total number of pixels is the same. A
+    uniformly offset dither sequence can be extracted from a well-resolved image that is drawn by
+    convolving the surface brightness profile explicitly with the native pixel response and setting
+    a lower sampling scale (or higher sampling rate) using the `pixel_scale' argument in drawImage()
+    routine and setting the `method' parameter to `no_pixel'.
+
+    Here is an example script using this routine:
+
+    Interleaving four Gaussian images
+    ---------------------------------
+
+        >>> n = 2
+        >>> gal = galsim.Gaussian(sigma=2.8)
+        >>> gal_pix = galsim.Convolve([gal,galsim.Pixel(scale=1.0)])
+        >>> img = gal_pix.drawImage(gal_pix,scale=1.0*n,method='no_pixel')
+        >>> im_list, offsets = galsim.utilities.deInterleaveImage(gal_pix,N=n)
```

Comment:
Also I'm a little confused about the philosophy of how this was set up.  If you are setting this up as a  way of drawing a single image that is sampled more finely than the native pixel scale, and then recovering multiple images at the native pixel scale by de-interleaving, then I would think you would have a pixel response that corresponds to a **larger** physical scale than the drawn image.  (i.e., your native pixel scale is x but you draw it into a pixel scale that is x/2, then de-interleave.)   Right now you have a 1 arcsec pixel response, then draw into an image with 2 arcsec pixels, so you're sampling *less* finely; what is the use case for that?